src/common/Common.ts

```diff
@@ -75,3 +75,43 @@ export class CommandLineUtils {
         }
     }
 }
+
+// tslint:disable-next-line: max-classes-per-file
+export class Version {
```

Comment:
I'm not against rolling our own per se. But I'm also cognizant of the fact that this is already a [solved problem](https://www.npmjs.com/package/semver). Since we're keeping it simple, I'm probably in favor of doing our own, hopefully less maintenance.

Reply:
yes, SemVer seemed like overkill to me so I opted for doing our own.